hdfs/env_hdfs.h

```diff
@@ -210,7 +207,7 @@ class HdfsEnv : public Env {
                            remaining.substr(0, rem));
 
     tPort port;
-    port = atoi(portStr.c_str());
+    port = static_cast<tPort>(atoi(portStr.c_str()));
```

Comment:
Nit:
```
tPort port = static_cast<tPort>(atoi(portStr.c_str()));
```

Reply:
Also, can an illegal value of `portStr` lead to unhandled error?

Reply:
I guess we can leave this as is.

Reply:
thanks, updated.